services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -343,7 +343,11 @@ export const SideBar: React.FC<{ className?: string; toggleSidebar: () => void }
                 .catch((e) => {
                     // eslint-disable-next-line no-console
                     console.error('error in batch request: ', e);
-                    showSnackbarError('Actions were not applied. Please try again');
+                    if (e.message.includes('PermissionDenied')) {
```

Comment:
```suggestion
                    const GrpcErrorPermissionDenied = 7;
                    if (e.code === GrpcErrorPermissionDenied) {
```

Reply:
please do not rely on the text in the message. That's what the code is made for.